### PR TITLE
[TASK] Add support for `052.001.08` and `053.001.08`

### PR DESCRIPTION
**Pull request by sbuerk**
* added xsd files to assets folder:

  - assets/camt.052.001.08.xsd
  - assets/camt.053.001.08.xsd

* added support for 052.001.08:

  - added message-format and decoder for 052.001.08
  - added camt052.v8.xml test file
  - added message v8 to EndToEnd tests
  - added 052.08 message to default config and adjusted default config test
  - added json file for regression test

* added support for 053.001.08:
  - added message-format for 053.001.08
  - added camt053.v8.xml test file
  - added 053.08 message to default config and adjusted default config test
  - added json file for regression test

* documented as supported in README.md

>> Same read capabilities are ensured like the versions
   before. Some new entities and attributes are missing
   and can be implemented in a dedicated change.

Resolves: #142
Related: #124
Related: #126

### CODE REVIEW

**Comment by sbuerk** [comment]
Hope, this is okay so far. If I missed something, let me know.

**Comment by sbuerk** [comment]
@PowerKiKi  Any change that this get's reviewed the next days and eventually a new version released ?

It's just for the planning, as we need to backport this library afterwards to be PHP5.6 compat again - but we have a deadline given by the bank.

Any thing I missed or need to change ?

**Comment by sbuerk** [comment]
ping

File: test/data/camt052.v6.xml
Comment (frederikbosch) [inline test/data/camt052.v6.xml:1]: Why did you change this file?

Comment (frederikbosch) [inline test/data/camt052.v6.xml:1]: Why did you change this file?

**Review by frederikbosch — COMMENTED** [review]
Just a minor question.

File: test/data/camt052.v6.xml
Comment (PowerKiKi) [reply test/data/camt052.v6.xml:1]: This is a whitespace-only changes that change indentation from 2 spaces to 4 spaces. IMHO this is fine.

Comment (frederikbosch) [reply test/data/camt052.v6.xml:1]: Right, but since there is no obvious reason to include a v6 change into this v8 support PR, I would suggest to leave the file as is.

**Review by PowerKiKi — APPROVED** [review]
LGTM, if @frederikbosch is OK with my last comment, I think we can merge and release.

**Comment by PowerKiKi** [comment]
Merged as 15f3d03f60ac88a0e49feec611d8343cd5aed4f9, which is exactly the same but with `test/data/camt052.v6.xml` reverted as suggested by Frederik.

Released as 2.5.0.
